src/getdependencies.ts

```diff
@@ -1,24 +1,44 @@
 import { error } from 'console';
+const readYaml =  require('read-yaml');
 
-let readJson;
+let readJson, parseYaml;
 let dependencies = [];
 
-export function getDependencies(repo: string, packageScope: string[]) {
+export function getDependencies(repo: string, packageScope: string[], repoMap: Map<string,string>) {
   try {
-    let path = '../repos/' + repo + '/package.json';
-
+    let path = '../' + repoMap.get(repo) + '/' + repo + '/package.json';
     dependencies = [];
```

Comment:
I think it would be good to declare this as `const` as well. In fact, it seems that adding `tslint` with recommended settings would be worthwhile for this project.